sunpy/map/mapbase.py

```diff
@@ -752,15 +749,71 @@ def date(self):
             # UTC is the FITS standard default
             timesys = self.meta.get('timesys', 'UTC')
 
+        return parse_time(time, scale=timesys.lower())
+
+    @property
+    def date_beg(self):
```

Comment:
I see no reason to tie ourselves to the weird short FITS names?

```suggestion
    def date_start(self):
```